src/routes.py

```diff
@@ -299,14 +303,24 @@ async def post_operation(
         entrypoint_name = operation["parameters"]["entrypoint"]
 
         try:
-            entrypoint = crud.get_entrypoint(db, str(contract.address), entrypoint_name)
+            entrypoint = crud.get_entrypoint(
+                db, str(contract.address), entrypoint_name)
             if not entrypoint.is_enabled:
                 raise EntrypointDisabled()
 
+            # Retrieving the user_id
+            # Check that if the user_id is among the 4 newest user
+            owner_address = contract.owner.address
+            user = crud.get_user_by_address(db, owner_address)
+            user_id = user.id
+            if not crud.check_user_is_new(db, user_id.id):
+                raise UserNotFound()
```

Comment:
So… if the user is not new, you're simply refusing the operation, regardless of the condition?